.travis.yml

```diff
@@ -1,17 +1,28 @@
 language: generic
 sudo: required
+addons:
+  homebrew:
+    update: true
+    packages:
+      - autoconf
+      - gnu-getopt
+      - gnu-sed
+      - gnu-tar
+      - libtool
+      - ninja
+      - pkg-config
+      - python3
+      - swig
+      - wget
 matrix:
   include:
     - os: osx
       osx_image: xcode11.3
       install:
-        - brew update
-        - brew upgrade
-        - brew install ninja autoconf libtool gnu-sed python3 wget pkg-config swig gnu-getopt gnu-tar
         - python3 -m pip install virtualenv meson===0.51.2
         - curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.42.0
       script:
-        - source $HOME/.cargo/env && ./tools/build.sh --clang --python-version 3.7 --enable-rust
+        - source $HOME/.cargo/env && ./tools/build.sh --clang --python-version $(python3 --version | sed -e 's/.* //' -e 's/.[0-9]$//') --enable-rust
```

Comment:
This does not work if the patch version is > 9

    echo "Python 3.6.10" | sed -e 's/.* //' -e 's/.[0-9]$//'
    3.6.

The regex should be tweaked or we can use

     python3 -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')"

Reply:
Tweaked, thanks